### PR TITLE
removed unnecessary sigmoid in classification.

### PR DESCRIPTION
**Pull request by t1t0n**
WeightedFocalLoss uses binary_cross_entropy_with_logits which expects inputs without sigmoid. From PyTorch documentation: "This loss combines a Sigmoid layer and the BCELoss in one single class.". [Source](https://pytorch.org/docs/stable/generated/torch.nn.BCEWithLogitsLoss.html#torch.nn.BCEWithLogitsLoss) So no need to add Sigmoid at the end of classification branch.

### CODE REVIEW

**Comment by meilfang** [comment]
Thank your for your correction and contribution.
